### PR TITLE
add license and notice to jars

### PR DESCRIPTION
**Pull request by pjfanning**
* based on what is in incubator-pekko repo

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,16 +1,201 @@
-This software is licensed under the Apache 2 license, quoted below.
+                                 Apache License
+                           Version 2.0, January 2004
+                        https://www.apache.org/licenses/
```
Comment (nvollmar) [inline LICENSE:3]: should use http

Comment (pjfanning) [reply LICENSE]: thanks - I copied the license from an out of date repo - fixed now

**Comment by pjfanning** [comment]
@nvollmar would you be able to re-review if you time?

**Review by nvollmar — APPROVED** [review]

File: NOTICE [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Copyright 2022, 2023 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (https://www.apache.org/).
+
+This product contains significant parts that were originally based on software from Lightbend (Akka <https://akka.io/>).
+Copyright (C) 2014 - 2016 Softwaremill [http://softwaremill.com]
+Copyright (C) 2016 - 2022 Lightbend Inc. [http://www.lightbend.com]
+
+Apache Pekko is derived from Akka 2.6.x, the last version that was distributed under the
```
[the remark this answers is not in the document]
Comment (pjfanning) [reply NOTICE]: thanks - I've fixed that now

**Review by nvollmar — APPROVED** [review]

**Pull request merged**